src/img/stval2reg.edn

```diff
@@ -6,9 +6,9 @@
 (def left-margin 100)
 (def right-margin 100)
 (def boxes-per-row 32)
-(draw-column-headers {:height 20 :font-size 18 :labels (reverse ["0" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "MXLEN-1"])})
+(draw-column-headers {:height 20 :font-size 18 :labels (reverse ["0" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "SXLEN-1"])})
 
 (draw-box "Addr"   {:span 32})
 
-(draw-box "MXLEN" {:span 32 :borders {}})
+(draw-box "SXLEN" {:span 32 :borders {}})
 ----
```

Comment:
This commit should really be separate as it's a fix to stval2 and not part of adding vstval2

Reply:
yeah fair point